app/models/room.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+#
+# Model to represent the lowest-level of facility information
+#
+# Attributes:
+#   - number: The number of the building, a string
+#     Required, must be unique within the suite the room belongs to
+#
+class Room < ApplicationRecord
+  belongs_to :suite
```

Comment:
We could also just have a `belongs_to :building` and write a validation to ensure that the room's building matches the suite's building?